index.html

```diff
@@ -0,0 +1,201 @@
+                        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+                        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+                    </p>
+                </div>
+            </div>
+
+            <div class="accordion-container">
+                <h3 id="accordion-header-2" class="accordion-header">
+                    <span>Portfolio</span>
+                </h3>
+                <div class="accordion-text-container">
+                    <p class="accordion-text">
+                        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+                        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+                    </p>
+                </div>
+            </div>
+
+            <div class="accordion-container">
+                <h3 id="accordion-header-3" class="accordion-header">
+                    <span>Contact us</span>
+                </h3>
+                <div class="accordion-text-container">
+                    <p class="accordion-text">
+                        "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                        labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+                        laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                        voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+                        non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+                    </p>
+                </div>
+            </div>
+
+        </section>
+
+        <!-- Form and actions -->
+        <section class="form-container">
+
+            <!--Fieldset tag -->
+            <fieldset>
+
+                <div class="form-heading">
+                    <h2>Get in touch!</h2>
+                    <p>Submit your contact information and we'll reach out.</p>
+                </div>
+
+                <div class="form">
+                    <form action="http://httpbin.org/anything" method="POST">
+
+                        <!-- 2 x text fields -->
+                        <div class="form-group">
+                            <label for="firstName"></label>
+                            <input id="firstname" type="text" name="firstName" placeholder="First name*" required />
```

Comment:
Nice to see some simple validation by using 'required', but to connect your label with for to the input-tag you need to use the id. Read some more here:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label